package-files/postinstall.sh

```diff
@@ -10,3 +10,8 @@ if [ -f "$HOMEDIR/connector.yaml" ]; then
     systemctl restart infra
   fi
 fi
+
+. /etc/os-release
+case $ID in
+  rhel) semodule -i /usr/share/infra/selinux/infra.pp ;;
```

Comment:
This may be unnecessary but where the source code of this file is pretty straightforward I wonder if we should just put the whole thing in a comment about this line in the script. 

Ex:
```
# **infra.pp**
# module infra 1.0;
# require {
#         type http_port_t;
#         type sshd_t;
#         class tcp_socket name_connect;
# }
# allow sshd_t http_port_t:tcp_socket name_connect;
rhel) semodule -i /usr/share/infra/selinux/infra.pp ;;
```

Just an idea though.